src/celengine/shadermanager.cpp

```diff
@@ -3415,13 +3417,18 @@ CelestiaGLProgram::attribIndex(const std::string& paramName) const
     return glGetAttribLocation(program->getID(), paramName.c_str());
 }
 
+void
+CelestiaGLProgram::initCommonParameters()
+{
+    ModelView = mat4Param("ModelView");
+    Projection = mat4Param("Projection");
```

Comment:
could you rename these variables to `ModelViewMatrix` and `ProjectionMatrix`?

Reply:
done.